### PR TITLE
Add new AppID for iOS 14 Home Widget

### PR DESCRIPTION
**Pull request by AliSoftware**
@Gio2018 recently added the "Wordpress Home Widget Today" App ID to our dev portal, so I figured it would be worth adding it in the list for our lanes managing profiles.

~🚧  This seems to currently fail though, because there's no InHouse profile with that AppID yet:~

> ~No matching provisioning profiles found for 'InHouse_org.wordpress.alpha.WordPressHomeWidgetToday.mobileprovision'
> A new one cannot be created because you enabled `readonly`~

[EDIT: Profiles have now been created by Jeremy, see below]

### CODE REVIEW

**Review by Gio2018 — COMMENTED** [review]
Hey @AliSoftware ! To give more context, I would expect that the current App Id will be actually used to release the new "Today Widget" (unless there are other issues that I am not aware of). There will be two more for two other widgets ("All Time" and "This Week") that will use the same naming convention. 
For this development we are using a feature branch (one per widget) since we can't use feature flags for widgets; so it will be merged only when the widget development has been completed.

**Comment by AliSoftware** [comment]
Ok so I guess that:
 - it's probably still too soon to merge that PR anyway until the feature lands
 - we'll need to push new commits to that PR later to add AllTime and ThisWeek widgets once the're ready before moving this PR forward anyway
 - But once this feature lands and development has been completed we can consider circle back to this PR to add it, right?

If you, can I ask you to ping back here in the future once you think the dev is completed to signal that we should be ready to move this forward? Thx 🙂

**Comment by jkmassel** [comment]
After some back-and-forth today, I ended up creating the profiles. We can either:

- Merge this into https://github.com/wordpress-mobile/WordPress-iOS/pull/15132

or

- Merge this into `develop` once https://github.com/wordpress-mobile/WordPress-iOS/pull/15132 lands

**Review by jkmassel — APPROVED** [review]

File: Scripts/fastlane/Fastfile
Comment (jkmassel) [inline Scripts/fastlane/Fastfile:12]: This is a very nice refactor! 😄

**Comment by AliSoftware** [comment]
I think it might make more sense to merge this into #15132 after all, That way potential additional changes in that draft PR over there could already add other BundleIDs for the other types of home widgets they plan to add in the future (ThisWeek, AllTimes).

_I see that the PR doesn't target `develop` yet but an intermediate branch likely used for incremental work until this is ready for develop, so there's likely gonna be more PRs targetting that `feature/ios-14-today-widget` branch before said branch ends up in develop anyway_

**Comment by AliSoftware** [comment]
@Gio2018 I changed the target branch of this PR to point to your #15132 draft PR. Feel free to approve and merge when ready and continue your work on your draft PR from there :)

**Review by Gio2018 — APPROVED** [review]
Thanks for this @AliSoftware !!

**Pull request merged**
